odoo/addons/base/migrations/12.0.1.3/noupdate_changes.xml

```diff
@@ -1,17 +1,174 @@
+  </record>
+  <record id="ms" model="res.country">
+    <field eval="1664" name="phone_code"/>
+  </record>
+  <record id="as" model="res.country">
+    <field eval="1684" name="phone_code"/>
+  </record>
+  <record id="ss" model="res.country">
+    <field eval="211" name="phone_code"/>
+  </record>
+  <record id="mf" model="res.country">
+    <field eval="590" name="phone_code"/>
+  </record>
+  <record id="cw" model="res.country">
+    <field eval="599" name="phone_code"/>
+  </record>
+  <record id="tf" model="res.country">
+    <field eval="262" name="phone_code"/>
+  </record>
+  <record id="gs" model="res.country">
+    <field eval="500" name="phone_code"/>
+  </record>
+  <record id="bs" model="res.country">
+    <field eval="1242" name="phone_code"/>
+  </record>
+  <record id="vc" model="res.country">
+    <field eval="1784" name="phone_code"/>
+  </record>
+  <record id="sj" model="res.country">
+    <field eval="47" name="phone_code"/>
+  </record>
+  <record id="um" model="res.country">
+    <field eval="699" name="phone_code"/>
+  </record>
+  <record id="bb" model="res.country">
+    <field eval="1246" name="phone_code"/>
+  </record>
+  <record id="fk" model="res.country">
+    <field eval="500" name="phone_code"/>
+  </record>
+  <record id="ai" model="res.country">
+    <field eval="1264" name="phone_code"/>
+  </record>
+  <record id="vg" model="res.country">
+    <field eval="1284" name="phone_code"/>
+  </record>
+  <record id="jm" model="res.country">
+    <field eval="1876" name="phone_code"/>
+  </record>
+  <record id="pr" model="res.country">
+    <field eval="1939" name="phone_code"/>
+  </record>
+  <record id="hm" model="res.country">
+    <field eval="672" name="phone_code"/>
+  </record>
+  <record id="ky" model="res.country">
+    <field eval="1345" name="phone_code"/>
+  </record>
+  <record id="vi" model="res.country">
+    <field eval="1340" name="phone_code"/>
+  </record>
+  <record id="io" model="res.country">
+    <field eval="246" name="phone_code"/>
+  </record>
+  <record id="ps" model="res.country">
+    <field eval="970" name="phone_code"/>
+  </record>
+  <record id="uk" model="res.country">
+    <field eval="44" name="phone_code"/>
+  </record>
+  <record id="gd" model="res.country">
+    <field eval="1473" name="phone_code"/>
+  </record>
+  <record id="sg" model="res.country">
+    <field name="vat_label">GST No.</field>
+  </record>
+  <record id="je" model="res.country">
+    <field eval="44" name="phone_code"/>
+  </record>
+  <record id="cx" model="res.country">
+    <field eval="61" name="phone_code"/>
+  </record>
+  <record id="ag" model="res.country">
+    <field eval="1268" name="phone_code"/>
+  </record>
+  <record id="bl" model="res.country">
+    <field eval="590" name="phone_code"/>
+  </record>
+  <record id="bv" model="res.country">
+    <field eval="55" name="phone_code"/>
+  </record>
+  <!--
+  <record id="partner_root" model="res.partner">
```

Comment:
Why do you comment this? I see this important for following Odoo's paradigm

Reply:
I just commented it because I wasn't sure. I will put again now :)

Reply:
Travis failed because of decommenting this.
Error: "You cannot archive a contact linked to an internal user."

Reply:
Then you should do it by SQL.

Reply:
Or see if the problem is that the user is not inactive.

Reply:
The user (base.user_root) is active

Reply:
in v11, both partner and user were active. Is in v12 that the partner becomes inactive. I think the way to handle it it's to let the partner be active in this file but later put inactive with sql.

Reply:
OK, good for me that way.

Reply:
After some checking, I concluded that this should be commented instead, because the user_root/user_admin/partner_root/partner_admin thing is handled in the pre-migration and in the end-migration. It was done by Stefan.

Reply:
OK, then put these conclusions here as well for your future "you" and the rest of us.